### PR TITLE
feat: add google indexing troubleshooting

### PR DESCRIPTION
**Pull request by patrickhousley**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

Adding a troubleshooting document for the browser agent around the possible existence of 404 errors in customer's Google Search dashboards.

### CODE REVIEW

File: src/content/docs/browser/new-relic-browser/troubleshooting/google-indexing-unknown-paths.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## Solutions
+
+This is not an uncommon scenario for site administrators and is not just linked to the Browser Agent. There could be many causes for strings to appear in your site HTML that Google Search may perceive as a potential path within your site. See the below resources of other people having this same concern.
+
+- [How can I block Google from crawling things it thinks are URLs from __NEXT_DATA__?](https://stackoverflow.com/questions/75672103/how-can-i-block-google-from-crawling-things-it-thinks-are-urls-from-next-data)
+- [Google follows JavaScript string as relative path - produces 404 error](https://webmasters.stackexchange.com/questions/50848/google-follows-javascript-string-as-relative-path-produces-404-error)
+
+Available internet resources indicate these 404 errors will not affect your site ranking or indexing. They can be safely ignored. However, if you are still concerned, you can using the [Google Search Support Community](https://support.google.com/webmasters/community?hl=en) to get additional feedback and help.
```
Comment (metal-messiah) [inline src/content/docs/browser/new-relic-browser/troubleshooting/google-indexing-unknown-paths.mdx:26]:
```suggestion
Available internet resources indicate these 404 errors will not affect your site ranking or indexing. They can be safely ignored. However, if you are still concerned, you can reach out to the [Google Search Support Community](https://support.google.com/webmasters/community?hl=en) to get additional feedback and help.
```

File: src/content/docs/browser/new-relic-browser/troubleshooting/google-indexing-unknown-paths.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Cause
+
+The Browser Agent makes use of string literals in code but converts those to string concatenation during the build process to prevent possible issues with code that may wrap the loader string in a string literal. In some cases, this may lead to strings the start with a forward slash `/`. When Google indexes the page containing the agent loader script, it will store these strings as potential paths for your site and attempt to index them.
+
+## Solutions
+
+This is not an uncommon scenario for site administrators and is not just linked to the Browser Agent. There could be many causes for strings to appear in your site HTML that Google Search may perceive as a potential path within your site. See the below resources of other people having this same concern.
+
+- [How can I block Google from crawling things it thinks are URLs from __NEXT_DATA__?](https://stackoverflow.com/questions/75672103/how-can-i-block-google-from-crawling-things-it-thinks-are-urls-from-next-data)
```
Comment (metal-messiah) [inline src/content/docs/browser/new-relic-browser/troubleshooting/google-indexing-unknown-paths.mdx:23]: How stable are stackoverflow links?  Can we reasonably trust these won't go "dead"?

Comment (patrickhousley) [reply src/content/docs/browser/new-relic-browser/troubleshooting/google-indexing-unknown-paths.mdx:23]: I would not worry about it. SO is pretty stable and doesn't seem to ever delete anything.

**Review by metal-messiah — DISMISSED** [review]
Looks good from a Browser Agent perspective

**Comment by rhetoric101** [comment]
I just updated from develop because this was stalled.

**Review by homelessbirds — APPROVED** [review]

**Pull request merged**